internal/handlers/common/update.go

```diff
@@ -118,5 +122,77 @@ func UpdateDocument(doc, update *types.Document) (bool, error) {
 		}
 	}
 
-	return true, nil
+	return changed, nil
+}
+
+// ValidateUpdateQuery validates update statement.
+func ValidateUpdateQuery(update *types.Document) (err error) {
```

Comment:
This is a query actually. `ValidateUpdateOperators` might be the better name.

Reply:
not much change and it's not about operators, it's about the entire query document.

Reply:
thank you, changed.